entities/src/models.rs

```diff
@@ -33,11 +33,12 @@ impl UrlWithStatus {
         }
     }
     pub fn get_metadata_id(&self) -> Vec<u8> {
+        Self::get_metadata_id_for(&self.metadata_url)
```

Comment:
Cannot we inline the body of `get_metadata_id_for` here? Do we need a function that just calls a function?